### DISCUSSION TITLE
Add a FixtureConnector

### DISCUSSION THREAD

**Discussion by ZuluPro** [Archmonger/django-dbbackup#579] [Feature Requests]
The primary solution before solutions like DBBackup is `dumpdata`/`loaddata`. I don't like it because serialization/deserialization take time. But it can be useful to migrate DB across environment, example: a production in Postgres and testing in SQLite.

I think about a `FixtureConnector` allowing to dump and restore with Django built-in features and working with DBBackup mechanisms (encryption, compression, uploading).

**Comment by iosifnicolae2** [comment]
@ZuluPro https://github.com/django-dbbackup/django-dbbackup/pull/309 might be useful
